src/app/organizations/organization-members/organization-members.component.ts

```diff
@@ -125,4 +125,9 @@ export class OrganizationMembersComponent extends Base implements OnInit {
         }
       });
   }
+
+  getUserORCIDLink(organizationUser: OrganizationUser) {
+    const orcid = organizationUser.user.orcid;
```

Comment:
If you put a console.log in here, I think you'll see this method is getting invoked a lot. If so, don't use a method in the HTML.

Reply:
I'm only seeing it get invoked once per user with an ORCID